pandas/io/excel/_pyxlsb.py

```diff
@@ -0,0 +1,65 @@
+from typing import List
+
+from pandas.compat._optional import import_optional_dependency
+
+from pandas._typing import FilePathOrBuffer, Scalar
+
+from pandas.io.excel._base import _BaseExcelReader
+
+
+class _PyxlsbReader(_BaseExcelReader):
+    def __init__(self, filepath_or_buffer: FilePathOrBuffer) -> None:
```

Comment:
Stylistically can remove the `-> None` here

Reply:
Done in the latest commit